pandas/tests/frame/indexing/test_set_value.py

```diff
@@ -26,10 +26,7 @@ def test_set_value_resize(self, float_frame):
         assert float_frame._get_value("foobar", "qux") == 0
 
         res = float_frame.copy()
-        with tm.assert_produces_warning(
-            FutureWarning, match="Setting an item of incompatible dtype"
-        ):
-            res._set_value("foobar", "baz", "sam")
+        res._set_value("foobar", "baz", "sam")
```

Comment:
this is like the reported issue, and probably shouldn't have been warning in the first place